Lines discovered around Pleasantville, New Argoth, and Wellspring

alienacorn opened the pull request.

alienacorn committed:
Expanded xrange and zrange to [-3000,3000]

alienacorn committed:
Expanded xrange and zrange to [-3000,3000]

alienacorn committed:
Added Pleasantville connections

alienacorn committed:
Added Pleasantville connections

alienacorn committed:
Added Whore Island and Monkeh Island rails.

alienacorn committed:
Added Whore Island and Monkeh Island rails.

alienacorn committed:
Added line between Wellspring and Spawn (SE Terminal)

alienacorn committed:
Added line between Wellspring and Spawn (SE Terminal)

alienacorn committed:
Added Wellspring/New Athens/New Argoth rail.

alienacorn committed:
Added Wellspring/New Athens/New Argoth rail.

alienacorn committed:
Added notes to staions without surface access.

alienacorn committed:
Added notes to staions without surface access.

alienacorn committed:
Added more lines near Pleasantville.

alienacorn committed:
Added more lines near Pleasantville.

alienacorn committed:
Added Hall of Pillars rail.

alienacorn committed:
Added Hall of Pillars rail.

Commit:
Made Argoth/Wellspring connect through New Athens directly.

This is not actually correct, but it makes for a better subway map without losing too much detail.

Commit:
Made Argoth/Wellspring connect through New Athens directly.

This is not actually correct, but it makes for a better subway map without losing too much detail.

alienacorn committed:
Added lines.

alienacorn committed:
Added lines.

alienacorn committed:
Updated rails around the northeast junction.

alienacorn committed:
Updated rails around the northeast junction.

alienacorn committed:
Added missing colon.

alienacorn committed:
Added missing colon.

alienacorn committed:
Merge branch 'patch-1'

alienacorn committed:
Added rails in the south.

alienacorn committed:
Added rails in the south.

tdaede committed:
Add new script generate-nodes.py, which outputs a json file containing a list of stations and which destinations they connect to. This format is more convenient for automatic pathfinding.

tdaede committed:
Add new script generate-nodes.py, which outputs a json file containing a list of stations and which destinations they connect to. This format is more convenient for automatic pathfinding.

tdaede committed:
Add new script generate-nodes.py, which outputs a json file containing a list of stations and which destinations they connect to. This format is more convenient for automatic pathfinding.

tdaede committed:
Add proper loop rail traversal and fix twoway rails.

tdaede committed:
Add proper loop rail traversal and fix twoway rails.

tdaede committed:
Route planning html page! Automatically calculates routes up to 8 stops, counting the start and end points. Results are given in a list sorted by the number of stops.

Modify generate-nodes.py to support a hacky javascript file output mode as well, required by route.html right now.

tdaede committed:
Route planning html page! Automatically calculates routes up to 8 stops, counting the start and end points. Results are given in a list sorted by the number of stops.

Modify generate-nodes.py to support a hacky javascript file output mode as well, required by route.html right now.

tdaede committed:
Add a bit of labels to the html page so people know what it even is.

tdaede committed:
Add a bit of labels to the html page so people know what it even is.

tdaede committed:
Merge remote-tracking branch 'alienacorn/master'

Conflicts:
	generate-nodes.py

tdaede committed:
Add partial cycle detection. This should cut down on the number of silly results, though many of the paths still have cycles. Because we care only about the shortest route, I'm not sure that it's faster to do a complete check.
Also, increase the search depth by one. PvE is ridiculous.

bnason committed:
Added ability to define rail turns

bnason committed:
Added West Rail line and turns to 850 and y70 loops

leafstorm committed:
Merge pull request #4 from Klibbnisse/patch-1

Added West Portal, QN junction, Unknown Station to the Y70 outer loop.

leafstorm commented:
While the routes HTML page is nice, I'm not quite ready to merge that in yet - I would prefer for the JSON file to be generatable from the Ruby script, and have more detailed information. Can you reduce the range of the pull request to everything before 958277d?

tdaede committed:
Add route distance calculation and sorting based on that.

Klibbnisse committed:
Adding some stations. No lines tough, I haven't really gotten the hang of the markup just yet. 

Cloud City is connected to North Portal, Montpellier Central is connected to Montpellier->Montpellier Central->North Portal (in that order, there's also a line Pico->Montpellier that's separate from the Y70. Montpellier should really be renamed "Montpellier Y70" or something.

We also have a line Pico->Backcountry Station->World's End Outpost. This is a both-way underground track.

Then we have a line going from Southwest 3000->Northwest 3000->World's End Outpost. It's probably longer but that's all I have explored right now.

tdaede committed:
hard-hitting CSS action

tdaede committed:
fix silly off by one error in distance calculation

pixelcort committed:
Specify which PvE Rail Network this is.

pixelcort committed:
Rename to better identify which PvE server this is referring to.

pixelcort committed:
Update README to refer to new filename for yaml file.

alienacorn committed:
Added option to supress JSON output.

alienacorn committed:
Ignore corners in route planner.

alienacorn committed:
Merge branch 'master' of https://github.com/bnason/railgen

Conflicts:
	generate-nodes.py
	pve-6-rails.yaml
	route.html

alienacorn committed:
Merge in Klibbnisse's additions.

Conflicts:
	pve-6-rails.yaml

alienacorn committed:
Added Klibbnisse's discovered lines.

alienacorn committed:
Updated .gitignore

alienacorn committed:
Added build script.

alienacorn committed:
Merge branch 'master' of https://github.com/pixelcort/railgen

Conflicts:
	reddit-pve-6-rails.yaml

alienacorn commented:
As you can tell by the title of this request, I did not expect for all these commits to be included too.
